sphinx/quickstart/Makefile.new_t

```diff
@@ -0,0 +1,21 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
I think it is better if template files are in `sphinx/templates` directory with other template files if needed.